server/graphql/general/src/queries/store_preference.rs

```diff
@@ -20,6 +20,16 @@ impl StorePreferenceNode {
     pub async fn pack_to_one(&self) -> &bool {
         &self.store_preference.pack_to_one
     }
+    pub async fn use_authorisation_for_customer_requisitions(&self) -> &bool {
```

Comment:
Hmm, I don't think it would be needed for front end, but I guess no probs exposing extra field in API

Reply:
agree, not required now. I've taken it out - we can re-introduce if required.